README.md

```diff
@@ -196,8 +198,13 @@ export class MySequence implements SequenceHandler {
         permissions, // do authUser.permissions if using method #1
         request,
       );
+
+      const isCustomAccessAllowed: boolean = await this.checkCustomAuthorisation(
```

Comment:
what benefit is this new fn giving in this new context. Its exactly same as checkAuthorisation. Whats different its doing ? what's the purpose ?